ansible/roles/operator-pipeline/templates/openshift/tasks/preflight-trigger.yml

```diff
@@ -46,188 +46,166 @@ spec:
-              OCP_VER="${DPTP_VER}"
-          else
-              OCP_VER="${CP_VER}"
-          fi
-
-          if [ "$(params.ocp_version)" == "4.6" ] || [ "$(params.ocp_version)" == "4.7" ]; then
-              JOB_SUFFIX="aws"
-          else
-              JOB_SUFFIX="claim"
-          fi
-
-          set -e
-
-          if [ -f "$(workspaces.credentials.path)/project_dockerconfigjson.gpg.b16" ]
-          then
-              PFLT_DOCKERCONFIG="$(cat "$(workspaces.credentials.path)/project_dockerconfigjson.gpg.b16")"
-              export PFLT_DOCKERCONFIG
-
-              preflight-trigger \
-          -job-name "periodic-ci-redhat-openshift-ecosystem-preflight-ocp-${OCP_VER}-preflight-common-${JOB_SUFFIX}" \
-          -job-config-path "release/ci-operator/jobs/redhat-openshift-ecosystem/preflight/redhat-openshift-ecosystem-preflight-ocp-${OCP_VER}-periodics.yaml" \
-          -prow-config-path release/core-services/prow/02_config/_config.yaml \
-          -ocp-version "${OCP_VER}" \
-          -pflt-docker-config "${PFLT_DOCKERCONFIG}" \
-          -pflt-artifacts artifacts \
-          -pflt-log-level "$(params.log_level)" \
-          -pflt-index-image "$(params.bundle_index_image)" \
-          -test-asset "$(params.bundle_image)" \
-          -asset-type "$(params.asset_type)" \
-          -output-path prowjob-base-url
-          else
-              preflight-trigger \
-          -job-name "periodic-ci-redhat-openshift-ecosystem-preflight-ocp-${OCP_VER}-preflight-common-${JOB_SUFFIX}" \
-          -job-config-path "release/ci-operator/jobs/redhat-openshift-ecosystem/preflight/redhat-openshift-ecosystem-preflight-ocp-${OCP_VER}-periodics.yaml" \
-          -prow-config-path release/core-services/prow/02_config/_config.yaml \
-          -ocp-version "${OCP_VER}" \
-          -pflt-artifacts artifacts \
-          -pflt-log-level "$(params.log_level)" \
-          -pflt-index-image "$(params.bundle_index_image)" \
-          -test-asset "$(params.bundle_image)" \
-          -asset-type "$(params.asset_type)" \
-          -output-path prowjob-base-url
-          fi
       workingDir: $(workspaces.output.path)
     - env:
         - name: PIPELINE_GPG_DECRYPTION_PRIVATE_KEY
           value: /etc/gpg-key/$(params.gpg_decryption_private_key_secret_key)
         - name: PIPELINE_GPG_DECRYPTION_PUBLIC_KEY
           value: /etc/gpg-key/$(params.gpg_decryption_public_key_secret_key)
-      image: "$(params.fedora_image)"
+      image: $(params.preflight_trigger_image)
       name: get-prowjob-artifacts-and-decrypt
       resources: {}
+      script: >
+        #!/usr/bin/env bash
+
+
+        if [ "$(params.skip)" == "true" ]
+        then
+          echo "Preflight testing has been skipped"
+          echo -n "none" > "$(results.log_output_file.path)"
+          echo -n "none" > "$(results.result_output_file.path)"
+          echo -n "none" > "$(results.artifacts_output_dir.path)"
+          exit 0
```

Comment:
If we're gonna echo anything, maybe `skipped`?

Reply:
This is from the existing task not sure if "none" is being used on purpose.